Package.swift

```diff
@@ -55,7 +55,7 @@ let package = Package(
             "OhhAuth",
             "SwiftSoup",
             .product(name: "SwiftPMPackageCollections", package: "SwiftPM")
-        ]),
+        ], exclude: [ "Core/SearchFilter/README.md" ]),
```

Comment:
Feedback from Dave:

Change this README some more, and make it user-facing in the FAQs